test/IRGen/jit_metadata_table.swift

```diff
@@ -0,0 +1,9 @@
+// RUN: %target-swift-frontend %s -emit-ir -use-jit | %FileCheck %s
+
+// Check that we emit all local decls, not just the first one.
```

Comment:
I'm confused by this comment.

Reply:
🤦 sorry, this is a comment of the test I copied from.